examples/dapperish.go

```diff
@@ -12,7 +12,9 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/opentracing/api-golang/examples/dapperish"
+	// `go imports` was stumbling over dapperish since it's both a package name
+	// and a source file.
+	dapperish "github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
There is no `go imports` command, did you mean `goimports` ?

Also, `goimports` only looks at `package` declarations -- not source file names..